src/models/constants.ts

```diff
@@ -0,0 +1,2 @@
+export const GRID_SIZE = 8;
+export const PADDING_SIZE = 25;
```

Comment:
If these are semantically grouped then it might be worth having an enum to leverage the editor (for example typing the enum name then being able to press crtrl+space to see the options).

```typescript
enum MapItemParams {
  GRID_SIZE = 8,
  PADDING_SIZE = 25
}
```